hw/ip/entropy_src/data/entropy_src.hjson

```diff
@@ -374,12 +374,9 @@
           name: "FIPS_ENABLE",
           mubi: true,
           desc: '''
-                Setting this field to `kMultiBitBool4True` selects the FIPS/CC compliant mode (or short FIPS mode).
-                In this mode, the ENTROPY_SRC block can generate FIPS qualified entropy.
-                Additional requirements to generate FIPS qualified entropy are i) that at most one of the !!ENTROPY_CONTROL.ES_ROUTE and !!ENTROPY_CONTROL.ES_TYPE fields are set to `kMultiBitBool4True` but not both, and ii) that !!CONF.RNG_BIT_ENABLE is set to `kMultiBitBool4False`.
-
                 Setting this field to `kMultiBitBool4False` selects the boot-time / bypass mode in which the hardware conditioning is bypassed.
-
+                To not bypass the conditioner at most one of the !!ENTROPY_CONTROL.ES_ROUTE and !!ENTROPY_CONTROL.ES_TYPE fields can be set to `kMultiBitBool4True` but not both.
```

Comment:
The grammar here took me a few seconds. Maybe "The conditioner will only be used if exactly one of !!... and !!... is set to kMultiBitBool4True." ?

Reply:
I think I just shortened this sentence but I agree this could be a little clearer. I'll amend it.